packages/mono-repo-publish/src/bin/mono-repo-publish.ts

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as yargs from 'yargs';
+import * as core from '../main';
+
+interface Args {
+  'pr-url': string;
+  'dry-run': boolean;
+}
+
+const publishCommand: yargs.CommandModule<{}, Args> = {
+  command: '$0',
+  describe: 'publish packages affected by a pull request',
+  builder(yargs) {
+    return yargs
+      .option('pr-url', {
+        describe:
+          'the URL of the GH PR for submodules you wish to publish, e.g., https://github.com/googleapis/release-please/pull/707',
+        type: 'string',
+        demand: true,
+      })
+      .option('dry-run', {
+        describe: 'whether or not to publish in dry run',
+        type: 'boolean',
+        default: false,
+      });
+  },
+  async handler(argv) {
+    const appIdPath = process.env.APP_ID_PATH;
```

Comment:
What if, instead of using these as arguments, we grab them from a secret in repo-automaiton-bots, like the rest of our bots?

Reply:
Let's follow up on that feature outside of this PR. I did this part because of typing and flow (didn't want to mess with the error handling in `getOctokitInstance()`)